services/simcore/docker-compose.yml

```diff
@@ -343,6 +419,13 @@ services:
       placement:
         constraints:
           - node.labels.dasksidecar==true
+      resources:
+        reservations:
+          cpus: "0.1"
+          memory: "256M"
+        limits:
+          cpus: "2"
+          memory: "1G"
```

Comment:
also again wondering here. the dask-sidecar is probably not using a lot of cpu in itself. But, it runs computational service that potentially use the whole machines.